repositories_go.bzl

```diff
@@ -0,0 +1,197 @@
+# Copyright 2019 Google LLC
```

Comment:
Please consider renaming this file to just `repositories.bzl`, because this whole repo is about Go codebase (so _go suffix is redundant), but even if it wasn't bazel consumes its dependencies on as repositories "abstractions", so regardless of the language to build one repo as dependency in another repo, you just import whatever it takes to build that regardless of the language. 

For example the grpc-java respository has this file names as just [repositories.bzl](https://github.com/grpc/grpc-java/blob/master/repositories.bzl)

Reply:
Done.